lib/collection/request-body.js

```diff
@@ -214,8 +214,10 @@ _.assign(RequestBody.prototype, /** @lends RequestBody.prototype */ {
     toJSON () {
         var obj = PropertyBase.toJSON(this);
 
-        // make sure that file content is removed because it is non-serializable ReadStream
-        _.unset(obj, 'file.content');
+        // remove value from file param if it's empty or non-string (can be non-serializable ReadStream)
+        if (obj.file && obj.file.content && typeof obj.file.content !== 'string') {
```

Comment:
Do we not want to support for buffer data itself or readstream ?

Reply:
Any non-string value will be dropped.